assets/javascripts/discourse/components/chat-message-collapser.js

```diff
@@ -124,6 +150,24 @@ function hasUploads(uploads) {
   return uploads?.length > 0;
 }
 
+function imagePredicate(e) {
```

Comment:
Not really related to your PR but since the functions are not exported, is there a reason we choose to define the functions outside the component instead of declaring them as private functions in the component itself?

Reply:
Actually the last function in the file (`isCollapsible`) is exported.

Reply:
Minor but what do you think about `isImage` as the function name as compared to using `imagePredicate`?